modules/nf-core/nanoq/main.nf

```diff
@@ -0,0 +1,50 @@
+process NANOQ {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda "${moduleDir}/environment.yml"
+    container "${workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/nanoq:0.10.0--h031d066_2' :
+        'biocontainers/nanoq:0.10.0--h031d066_2'}"
+
+    input:
+    tuple val(meta), path(ontreads)
+    val(output_format) //One of the following: fastq, fastq.gz, fastq.bz2, fastq.lzma, fasta, fasta.gz, fasta.bz2, fasta.lzma.
+
+    output:
+    tuple val(meta), path("*.stats")                                                  , emit: stats
+    tuple val(meta), path("*_nanoq.${output_format}")                                                 , emit: reads
```

Comment:
Are these "reads" if they are fasta sequences?

Reply:
Yes, those reads could be `fasta` format.

Reply:
I just meant whether "reads" was a good name, as I assume that to be fastq files (but if it is fine that's not a problem, just asking the quesiton)